slash-commands/slash-commands.js

```diff
@@ -115,14 +149,23 @@ CandyShop.SlashCommands = (function(self, Candy, $) {
 	self.part = function() {
 		Candy.Core.Action.Jabber.Room.Leave(self.currentRoom());
 	};
+	
+	/** Function: leave
+	 * Exits the current chat room
+	 *
+	 */
+	self.leave = function() {
```

Comment:
Instead of creating a new function for that, I would merge both `leave` and `part` to the same one, and share the docstring as well.

Something like `self.part = self.leave = function() {`.